changes/ocd-4767.sql

```diff
@@ -0,0 +1,11 @@
+update openchpl.url_uptime_monitor_test
+set deleted = true,
+	last_modified_sso_user = '3438c418-50e1-70f0-4d80-804fb08dd7d5',
```

Comment:
This struck me as "different than what we usually do", but I guess it's not wrong? I just ran the report in DEV with SSO flag OFF and it seemed to work. Just want you to confirm it's correct considering the different flag states.